content/c-sharp/concepts/strings/terms/equals/equals.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.Equals()'
+Description: 'Determines whether two string objects have the same value or not.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Libraries'
+  - 'Arithmetic'
+  - 'Strings'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Equals()`** method determines whether two string objects have the same value or not.
+
+## Syntax
+
+```pseudo
+String.Equals(string1, string2)
```

Comment:
You also need to mention the syntax `string1,Equals(string2)` that you use in the example.